certs/api/endpoint.go

```diff
@@ -20,16 +20,11 @@ func issueCert(svc certs.Service) endpoint.Endpoint {
 		}
 		res, err := svc.IssueCert(ctx, req.token, req.ThingID, req.TTL)
 		if err != nil {
-			return certsRes{}, errors.Wrap(apiutil.ErrValidation, err)
+			return serialRes{}, errors.Wrap(apiutil.ErrValidation, err)
 		}
 
-		return certsRes{
-			CertSerial: res.Serial,
-			ThingID:    res.ThingID,
-			ClientCert: res.ClientCert,
-			ClientKey:  res.ClientKey,
-			Expiration: res.Expire,
-			created:    true,
+		return serialRes{
+			Serial: res.SerialNumber,
```

Comment:
We should return all the certificate data except the key. The same goes for Certs service endpoint.